Templates/Ros2FleetRobotTemplate/Template/Gem/enabled_gems.cmake

```diff
@@ -0,0 +1,40 @@
+# {BEGIN_LICENSE}
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+# {END_LICENSE}
+# This file is copied during engine registration. Edits to this file will be lost next
+# time a registration happens.
+
+set(ENABLED_GEMS
+    ${Name}
+    Atom
+    AudioSystem
+    AWSCore
+    CameraFramework
+    DebugDraw
+    EditorPythonBindings
+    EMotionFX
+    GameState
+    ImGui
+    LandscapeCanvas
+    LyShine
+    PhysX
+    PrimitiveAssets
+    PrefabBuilder
+    SaveData
+    ScriptCanvasPhysics
+    ScriptEvents
+    StartingPointInput
+    TextureAtlas
+    WhiteBox
+    DiffuseProbeGrid
+    Compression
+    ProteusRobot
+    WarehouseAssets
+    ROS2
```

Comment:
I would review this list and only default only the miinimal Gems that are necessary. For instance, I don't think the `GameState` or `SaveData` gem is needed for simulation

Reply:
I'm not sure if I removed all the unused Gems, but the list is shorter now :)

Reply:
Let us review it again